source/extensions/extensions_metadata.yaml

```diff
@@ -1729,13 +1729,20 @@ envoy.matching.inputs.transport_protocol:
   status: stable
   type_urls:
   - envoy.extensions.matching.common_inputs.network.v3.TransportProtocolInput
-envoy.matching.inputs.application_protocol:
+envoy.matching.inputs.endpoint_metadata:
   categories:
-  - envoy.matching.network.input
+  - envoy.matching.inputs
```

Comment:
nit: maybe call the category `envoy.matching.transport_socket.input`? That would be more consistent.